fastapi_websocket_rpc/rpc_channel.py

```diff
@@ -127,7 +132,8 @@ def __init__(self, methods: RpcMethodsBase, socket, channel_id=None, default_res
             methods (RpcMethodsBase): RPC methods to expose to other side
             socket: socket object providing simple send/recv methods
             channel_id (str, optional): uuid for channel. Defaults to None in which case a random UUID is generated.
-            default_response_timeout(int, optional) default timeout for RPC call responses. Defaults to None - i.e. no timeout
+            default_response_timeout(int, optional) default timeout for RPC call responses.
```

Comment:
I think we should change the default from None to a long timeout out (say 5 minutes) as a default that comes with potential memory leaks is a bad default.

Reply:
@jsdanielh Why does no timeout means a memory leak **after** the fix in `wait_for_response`?